lib/src/base-styles/_nx-typography.scss

```diff
@@ -28,8 +42,8 @@
 
 @font-face {
   font-family: "Open Sans";
-  src: url("../assets/fonts/open-sans-v17-latin-italic.woff") format("woff");
-  font-weight: normal;
+  src: url("../assets/fonts/open-sans-v18-latin-700italic.woff") format("woff");
```

Comment:
Seems we have a mix of v17 and v18 font files.  Should probably be consistent.

Reply:
> Seems we have a mix of v17 and v18 font files. Should probably be consistent.

I couldn't find any differences but sure: https://github.com/sonatype/sonatype-react-shared-components/pull/272/commits/14273e263e469f4ecd4af9f2296878ea58f87fa0 & https://github.com/sonatype/sonatype-react-shared-components/pull/272/commits/4837e6638527b1045fe5d477b778d0505325bb2e